kong/plugins/opentelemetry/migrations/init.lua

```diff
@@ -0,0 +1,3 @@
+return {
+  "001_331_to_332",
```

Comment:
this is the first version where we want to apply the fix (queue introduced in 3.3)